tools/doc-generator/parse/root_blocks.go

```diff
@@ -58,6 +59,11 @@ var (
 			StructType: reflect.TypeOf(frontend.CombinedFrontendConfig{}),
 			Desc:       "The frontend block configures the query-frontend.",
 		},
+		{
+			Name:       "query_scheduler",
+			StructType: reflect.TypeOf(scheduler.Config{}),
+			Desc:       "The query_scheduler configures query-scheduler component.",
```

Comment:
Sorry, I jus realized it's not consistent with the other descriptions. Let's make it consistent. I know it's a nit, but I've seen docs squad to be quite keen on that. Thanks!

```suggestion
			Desc:       "The query_scheduler block configures query-scheduler.",
```

Reply:
no worries, I should have looked as well. Honestly, they feel redundant to me, but not my area of expertise.

pushed the change in 2fd049d655ab7eee30cd87cdc6e7df29b2ba5d90

Reply:
Could you rebuild the doc? Then we merge it, promised! Actually I already push the auto-merge button ;)